models/importer.js

```diff
@@ -0,0 +1,42 @@
+import Promise from 'bluebird';
+import fs from 'fs';
+import Log from 'log';
+import config from '../config/config.json';
+
+const log = new Log();
+const readFileAsync = Promise.promisify(fs.readFile);
+
+export class Importer {
+  static import(path) {
+    return readFileAsync(path, 'utf8')
+      .then(content => Importer.csvToJson(content));
+  }
+
+  static importSync(path) {
+    if (fs.existsSync(path)) {
```

Comment:
Why do you have this check in importSync but there is no similar check in import? :)